src/main.js

```diff
@@ -868,17 +868,16 @@ textAngular.service('textAngularManager', ['taToolExecuteAction', 'taTools', 'ta
 		},
 		// this is used by taBind to send a key command in response to a special key event
 		sendKeyCommand: function(scope, event){
-			angular.forEach(editors, function(_editor){
```

Comment:
I think that is the correct behavior.  If i have two textAngular divs on the same page and I hit `Tab` in one of them (to trigger `indent`), why would I want that command to be sent to both divs?